### ISSUE TITLE
EPSModel has no attribute fwhm or sigma

### ISSUE THREAD

**Issue by SunilSimha** [astropy/photutils#804]
I'm using version 0.6 and there is no way I can use `IterativelySubtractedPSFPhotometry` because the `EPSModel` class has no attribute `fwhm`. When I tried using `IterativelySubtractedPSFPhotometry`, it gave an error that said:
```bash
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-81-0462a0dc654c> in <module>()
----> 1 result_tab=photometry(image=testimg)
      2 residual_image = photometry.get_residual_image()

~/anaconda3/lib/python3.6/site-packages/photutils/psf/photometry.py in __call__(self, image, init_guesses)
    185         """
    186 
--> 187         return self.do_photometry(image, init_guesses)
    188 
    189     def do_photometry(self, image, init_guesses=None):

~/anaconda3/lib/python3.6/site-packages/photutils/psf/photometry.py in do_photometry(self, image, init_guesses)
    660                                             gaussian_sigma_to_fwhm)
    661 
--> 662             output_table = self._do_photometry(['x_0', 'y_0', 'flux_0'])
    663         return output_table
    664 

~/anaconda3/lib/python3.6/site-packages/photutils/psf/photometry.py in _do_photometry(self, param_tab, n_start)
    701             apertures = CircularAperture((sources['xcentroid'],
    702                                           sources['ycentroid']),
--> 703                                          r=self.aperture_radius)
    704             sources['aperture_flux'] = aperture_photometry(
    705                 self._residual_image, apertures)['aperture_sum']

~/anaconda3/lib/python3.6/site-packages/photutils/aperture/circle.py in __init__(self, positions, r)
    122 
    123     def __init__(self, positions, r):
--> 124         if r < 0:
    125             raise ValueError('r must be non-negative')
    126 

TypeError: '<' not supported between instances of 'NoneType' and 'int'
```
Digging through the code revealed that r or `aperture_radius` wasn't defined properly in this block of code starting from line 235 of `photometry.py`

```python
if self.aperture_radius is None:
            if hasattr(self.psf_model, 'fwhm'):
                self.aperture_radius = self.psf_model.fwhm.value
            elif hasattr(self.psf_model, 'sigma'):
                self.aperture_radius = (self.psf_model.sigma.value *
                                        gaussian_sigma_to_fwhm)
```
Essentially, it skipped through this entire block without raising any flags because `EPSFModel` has no attribute by either name `sigma` or `fwhm`. Please fix this.

Thanks

**Comment by larrybradley** [comment]
Thanks, @SunilSimha.  This is the same issue reported in #728 (proposed fix in #740) and #849.

**Comment by larrybradley** [comment]
In the meantime, a workaround is to input an aperture_radius value (https://github.com/astropy/photutils/issues/728#issuecomment-428034935).
